plugin-script-python/build.gradle

```
@@ -0,0 +1,19 @@
+project.description = 'Execute Node scripts as part of Kestra data workflows.'
```

```suggestion
project.description = 'Execute Python scripts as part of Kestra data workflows.'
```

must be a copy-paste config issue :)